utils/src/main/java/org/corfudb/utils/lock/LockClient.java

```diff
@@ -61,6 +69,9 @@
     @Getter
     private UUID clientId;
 
+    private Lock lock;
```

Comment:
why are we making this a separate instance variable?  All locks are stored in the hashmap "locks"

Reply:
While lockclient suspend the lock, it will need to submit an event through the lock.